scripts/lib/encode.js

```diff
@@ -59,16 +59,60 @@ window.Encode.Unicode = {
 }
 
 window.Encode.Html = {
-  encode: value => {
+  encode_hex_mode: value => {
```

Comment:
If it is possible, I wish that function name should be named using camelCase.

> BTW, all methods added in this PR should follow it too, not only this one.

Reply:
Sure thing, I'll fix it!